optimum/habana/transformers/training_args.py

```diff
@@ -57,7 +57,6 @@
     "fp16_full_eval",
     "fp16_opt_level",
     "fsdp",
-    "half_precision_backend",  # not supported, Habana Mixed Precision should be used and specified in Gaudi configuration
```

Comment:
I think we should also remove `bf16` (cf. line 53) from this list and use it as the argument for enabling bf16 autocast.